dev-util/xxd/xxd-9.1.1652.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Standalone version of Vim's xxd"
+HOMEPAGE="https://www.vim.org https://github.com/vim/vim"
+SRC_URI="https://github.com/vim/vim/archive/v${PV}.tar.gz -> vim-${PV}.tar.gz"
```

Comment:
It updates significantly less frequently than the rest of vim. I recommend you use VIM_VERSION together with:
https://github.com/vim/vim/blob/306a138172c83cebf573b2ef7f56bf94d84429cb/src/xxd/xxd.c#L73

PV == 2025.08.24

Easier to explain to users than "new version alert! vim has new changes!"

Reply:
Makes sense - I just needed something to get started, but using xd's own version really is better. Thanks!